src/frontend/src/components/SearchPage/SearchResults.jsx

```diff
@@ -7,6 +7,7 @@ import { Container } from '@material-ui/core';
 import useSiteMetadata from '../../hooks/use-site-metadata';
 import Timeline from '../Posts/Timeline.jsx';
 import Spinner from '../Spinner';
+import NotFound from '../../images/NotFound.jpg';
```

Comment:
We should make sure to run this image (or whichever one we end up using) through https://tinyjpg.com/ or https://tinypng.com/. For the current one, it would reduce its size by 53%:
![image](https://user-images.githubusercontent.com/33902374/107048164-9e171400-6796-11eb-89ba-09b6d1f4bb35.png)
This would positively affect our site loading speed.